elife/base.sls

```diff
@@ -24,8 +22,12 @@ base:
             # provides add-apt-repository binary needed to install a new ppa easily
             # renamed in 18.04
             {% if osrelease in ['14.04', '16.04'] %}
+            # depends on py2
```

Comment:
so can't do anything about this one, right?

Reply:
right, not until we get the dozen projects on 16.04 on to 18.04: https://github.com/elifesciences/issues/issues/5095

even then ... well, cross that bridge when we get there